### ISSUE TITLE
Update needed to compile sheep_net on newer Linuxes

### ISSUE THREAD

**Issue by coderforlife** [cebix/macemu#120]
As of v4.11 of the Linux Kernel the function `signal_pending` is now in the header `linux/sched/signal.h>`. The following needs to be added near the top of `(SheepShaver|BasiliskII)/src/Unix/Linux/NetDriver/sheep_net.c` to fix the issue:

```C
#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,11,0)
#include <linux/sched/signal.h>
#endif
```

**Comment by rickyzhang82** [comment]
This fix works in 4.11.11-300.fc26.x86_64. Would you send the PR?

**Comment by coderforlife** [comment]
Sorry - I really messed up my fork of the code so there are a ton of extra messages there.
